app/main.py

```diff
@@ -1,3 +1,63 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[Distance, int]) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: Union[Distance, int]) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        else:
+            self.km += other
+            return self
+
+    def __mul__(self, other: Union[Distance, int]) -> Distance:
```

Comment:
```suggestion
    def __mul__(self, other: Union[float, int]) -> Distance:
```
Distance is not possible in __mul__ and __truediv__